Sources/Fluent/Exports.swift

```diff
@@ -1,5 +1,9 @@
+#if !BUILDING_DOCC
+
 @_exported import FluentKit
 
+#endif
+
 infix operator ~~
 infix operator =~
 infix operator !~
```

Comment:
These are technically re-export declarations as well and should also be within the exclusion. (Wish GH would let me select all five lines...)